tests/testmain.js

```diff
@@ -23,6 +23,41 @@ exports.testBasics = {
 	testMulDiv: function(test){
 		test.equals(Math.round(this.b.exec("345 / 23 * 124 / 41 * 12")), 544);
 		test.done();
+	},
+
+	testErrorData1: function(test){
+		try {
+			Math.round(this.b.execute("5 * FN(1"));
+			throw new Error('Should have thrown an error');
```

Comment:
Do we need this line to throw an error? Won't the `Math.round` call above throw an error itself? Wasn't that what this PR adds?

The same applies to the other 2 tests.

Reply:
Well never mind, I see it. We need that extra error throw here otherwise the `try` block will run without errors and the test will wrongly pass since the `catch` block which has assertions never runs. Neat trick. :)